shopware/Caddyfile

```diff
@@ -0,0 +1,45 @@
+DOMAIN {
+    gzip {
+        level 6
```

Comment:
Is this really the minimal Caddyfile needed for serving shopware? Like, is there a technical reason that shopware should use gzip and have level 6 compression?

(The goal of examples is to keep them as minimal and focused as possible. Otherwise people start wondering, "why is this required? works fine for me without it")